rq/queue.py

```diff
@@ -419,6 +423,33 @@ def __str__(self):
         return '<Queue {0!r}>'.format(self.name)
 
 
+class Batch(Queue):
+    def __init__(self, name='default', default_timeout=None, connection=None,
```

Comment:
I think it would be simpler to take a single `queue` argument instead of multiple arguments like `name` and `connection`. We'll be able to infer more information from the `queue` instance should we need more information about this queue in the future.

What do you think @jhorman?